routes/views/accounts/get/report.js

```
@@ -93,7 +101,15 @@ exports = module.exports = function(req, res) {
                     'statusStyle':statusStyle
                 });
             }
-            res.render('account/report', {flash: flash});
+            
+            fs.readFile('members/recent.json', 'utf8', function (err, data) {
+                try{
+                    locals.reportable_members = JSON.parse(data);
+                }
+                catch{}
```

Never empty catch. At least log something.